styles.css

```diff
@@ -0,0 +1,22 @@
+.button {
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  flex-direction: column-reverse;
+  align-self: center;
+}
+
+body {
+  background-color: darkblue;
+}
+
+.button {
```

Comment:
One issue here.. you've already modified the div class named "button"

Here, you're referencing another element by the exact same name. Even if it's not causing a problem yet, it will.

rename one or the other to something different in both `index.html` and `styles.css`

Reply:
ok